src/pages/items/armor/index.js

```diff
@@ -52,15 +55,18 @@ function Armor(props) {
     return [
         <Helmet key={'armor-table'}>
             <meta charSet="utf-8" />
-            <title>Armor chart</title>
+            <title>{t('Escape from Tarkov')} - {t('Armors')}</title>
```

Comment:
These changes (`Escape from Tarkov`) should even help improve SEO a bit which is fantastic!